cea/demand/airconditioning_model.py

```diff
@@ -9,11 +9,12 @@
     [1] Kämpf, Jérôme Henri
```

Comment:
@JIMENOFONSECA 
Full reporting a function used for testing the code and write intermediate results to .xls files. This can be activated/deactivated by setting `testing = False` in the globalvariables (like it used to be before).
Yes. The code is slower. Maybe because the speedy numba functions are not used anymore. I hope we can reestablish the speed. I'm working on that.